cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -219,7 +219,23 @@ spec:
           value: "true"
         - name: ROOK_CSI_ENABLE_GRPC_METRICS
           value: "true"
-
+        # - name: CSI_TOLERATION
+        #   value: "NoSchedule"
+        # (Optional) Ceph CSI toleration key. Set this to the key of the taint you want to tolerate
+        # - name: CSI_TOLERATION_KEY
+        #   value: "KeyOfTheTaintToTolerate"
+        # (Optional) Ceph CSI NodeAffinity.
+        # - name: CSI_NODE_AFFINITY
+        #   value: "role=storage-node; storage=rook, ceph"
+        # (Optional) CEPH CSI tolerations list. Put here list of taints you want to tolerate in YAML format.
+        # - name: CSI_TOLERATIONS
+        #   value: |
```

Comment:
This multi-line value works ok? I thought we had issues when trying it previously with other tolerations in operator.yaml so had to collapse to a single line.

Reply:
i referred agent toleration for this one, do I collapse to a single line

Reply:
If the YAML structure of `Value` for `CSI_TOLERATIONS` works as expected, we could possibly remove all other ENVs for CSI toleration and achieve the same effect with this one variable.
i.e. allow 
- Toleration Key only (and tolerate all Effects)
- Toleration Effect only (and tolerate all keys with)
- All of Key, Operator and Effect.

Reply:
Yes, that would be convenient if we could just deserialize these properties similarly to how we deserialize the `placement` struct from cluster.yaml. It would give all the flexibility we need.